subsys/bluetooth/host/hci_core.c

```diff
@@ -2534,6 +2534,12 @@ static const struct event_handler meta_events[] = {
 #endif /* CONFIG_BT_CONN */
 #endif /* defined(CONFIG_BT_PER_ADV_SYNC) */
 #endif /* defined(CONFIG_BT_EXT_ADV) */
+#if defined(CONFIG_BT_CTLR_SCAN_REQ_NOTIFY)
+#if defined(CONFIG_BT_BROADCASTER)
+	EVENT_HANDLER(BT_HCI_EVT_LE_SCAN_REQ_RECEIVED, bt_hci_le_scan_req_received,
+		      sizeof(struct bt_hci_evt_le_scan_req_received)),
+#endif
+#endif /* defined(CONFIG_BT_CTLR_SCAN_REQ_NOTIFY) */
```

Comment:
There is already a EVENT_HANDLER for BT_HCI_EVT_LE_SCAN_REQ_RECEIVED upstream, the code here seems redundant.

Reply:
Thanks for spotting this, it messed up with my different versions and missed to remove this. Fixed, working to the rest.